### PR TITLE
build ts files to js first before running script/test

### PR DESCRIPTION
**Pull request by udit-gulati**

### CODE REVIEW

File: packages/junokit/src/builtin-tasks/run.ts [hunk 1]
```diff
@@ -46,6 +47,10 @@ async function executeRunTask (
 ): Promise<any> {
   const logDebugTag = "junokit:tasks:run";
 
+  // build the ts scripts/test to js first
+  // path dependent, might find a better soln later
+  await buildTsScripts();
```
Comment (amityadav0) [inline packages/junokit/src/builtin-tasks/run.ts:52]: This will be really slow if there are multiple scripts and user is trying to run only one script.

Comment (amityadav0) [reply packages/junokit/src/builtin-tasks/run.ts:52]: One way would be run `yarn run build:watch`

Comment (amityadav0) [reply packages/junokit/src/builtin-tasks/run.ts:52]: I think we should include it in a tutorial

**Review by coco-sha — APPROVED** [review]
Looks good

**Pull request merged**